### DISCUSSION TITLE
Why did raspberry pi's dailyBild stop?

### DISCUSSION THREAD

**Discussion by aXdey65i** [elementary/triage#612] [Q&A] [answered]
The elementary OS raspberry pi version of daily Bild has been stopped since October 29. Is there no intention to restart? Please tell me.
![IMG_0690](https://github.com/user-attachments/assets/460be8b9-761f-479b-a21e-a6658e6741c0)

**Comment by ryonakano** [comment]
This should be because the image build for Raspberry Pi is failing at the moment. I've opened https://github.com/elementary/os/issues/727 and working on a fix, so please be patient.

**Reply by ryonakano** [reply, marked as answer]
@aXdey65i We worked on the build error and now the image for Raspberry Pi based on OS 8 Daily is available on the website.
